docs/_documentations/workingwithtemplates.md

```diff
@@ -14,23 +14,26 @@ parent: usingcodewind
+Template management in Codewind is provided by the **Template Source Manager**. To open the **Template Source Manager**, right-click **Local** > click **Manage Template Sources**. The **Template Source Manager** appears. 
 
-Codewind templates are stored in the [codewind-resources/codewind-templates](https://github.com/codewind-resources/codewind-templates) GitHub repository. Standard Codewind Templates, the default set of templates from Kabanero Collections, and Appsody Stacks to develop applications with sharable technology stacks, are already included in Codewind and you can refer to those as examples.
+Codewind templates are stored in the [codewind-resources/codewind-templates](https://github.com/codewind-resources/codewind-templates)GitHub repository. Three examples are included in Codewind for your reference: 
+1. Standard Codewind Templates.
+2. Default templates from Kabanero Collections.
+3. Appsody Stacks to develop applications with sharable technology stacks. 
 
 Use the **Template Source Manager** to perform the following actions:
 1. To add a new template source to the table, click **Add new**. For more information, see [Adding your own template sources to Codewind](#adding-your-own-template-sources-to-codewind).
 2. To remove non-default template sources, click the trash icon. 
-3. To enable a template source to have its templates appear in the **Create Project** wizard so that projects of that style can be added to Codewind, toggle the `Enabled` slide to on. For example, enable at least one Appsody-style source before trying to add an Appsody project.
-4. To disable a set of templates so they do not appear in the **Create Project** wizard, toggle the `Enabled` slide to off.
+3. To enable a template source to have its templates appear in the **Create Project** wizard so those style projects can be added to Codewind, toggle the `Enabled` slide to `On`. For example, enable at least one Appsody-style source before adding an Appsody project.
+4. To disable a set of templates so they do not appear in the **Create Project** wizard, toggle the `Enabled` slide to `Off`.
 
 ## Adding your own template sources to Codewind
 
 Add your own template sources to use Codewind with the framework of your choice. 
-1. Ensure your template source has an `index.json` file containing information about your new templates in the same format as, for example, [https://github.com/kabanero-io/codewind-templates/blob/master/devfiles/index.json](https://github.com/kabanero-io/codewind-templates/blob/master/devfiles/index.json).
+1. Ensure your template source has an `index.json` file containing information about your new templates in the same format as, [https://github.com/kabanero-io/codewind-templates/blob/master/devfiles/index.json](https://github.com/kabanero-io/codewind-templates/blob/master/devfiles/index.json).
```

Comment:
I think this step would be clearer if it were broken into shorter sentences. Maybe...`Ensure your template source has an `index.json` file containing information about your new templates. Follow the same format as the `index.json` file found in [codewind-templates/devfiles](https://github.com/kabanero-io/codewind-templates/blob/master/devfiles/index.json).`